fmatch/matcher.py

```diff
@@ -19,23 +19,30 @@ class Matcher:
     """Matcher"""
 
     def __init__(
-        self, index="perf_scale_ci", level=logging.INFO, ES_URL=os.getenv("ES_SERVER")
+        self, index="ospst-perf-scale-ci",
+        level=logging.INFO,
+        ES_URL=os.getenv("ES_SERVER"),
+        verify_certs=True
     ):
         self.index = index
         self.es_url = ES_URL
         self.search_size = 10000
         self.logger = logging.getLogger("Matcher")
         self.logger.setLevel(level)
         handler = logging.StreamHandler(sys.stdout)
-        handler.setLevel(level)
-        formatter = logging.Formatter(
-            "%(asctime)s - %(name)s - %(levelname)s - %(message)s"
-        )
-        handler.setFormatter(formatter)
-        self.logger.addHandler(handler)
+        # Check if handlers are already added to avoid duplication
```

Comment:
thinking we should keep this change as part of the separate pr [here](https://github.com/cloud-bulldozer/py-commons/pull/22/files)

Reply:
Yes it is handled in a seperate PR, added this to avoid confusion. I can roll-back a commit.